monai/handlers/mean_dice.py

```diff
@@ -72,13 +72,13 @@ def reset(self) -> None:
         self._num_examples = 0
 
     @reinit__is_reduced
-    def update(self, output: Sequence[Union[torch.Tensor, dict]]) -> None:
+    def update(self, output: Sequence[torch.Tensor]) -> None:
         if not len(output) == 2:
             raise ValueError("MeanDice metric can only support y_pred and y.")
         y_pred, y = output
         score = self.dice(y_pred, y)
         assert self.dice.not_nans is not None
-        not_nans = self.dice.not_nans.item()
+        not_nans = cast(int, self.dice.not_nans.item())
```

Comment:
this could be the actual casting `int(self.dice.not_nans.item())`

Reply:
changed